src/main/kotlin/dto/CellDto.kt

```diff
@@ -4,7 +4,7 @@ import domain.Cell
 
 @JvmInline
 value class CellDto(private val cell: Cell) {
-    override fun toString(): String = if (cell.isMine()) MINE else CLOSED
+    override fun toString(): String = if (cell.hasMine()) MINE else cell.mineNumber().toString()
```

Comment:
```suggestion
    override fun toString(): String = 
        if (cell.hasMine()) {
            MINE
        } else {
            cell
                .mineNumber()
                .toString()
        }
```
함수 체이닝 컨벤션을 적용해주시면 좋을 것 같아요